keyvalues/scripts/weapons/mp_weapon_r97.txt

```diff
@@ -111,13 +111,13 @@ WeaponData
 		"aimassist_adspull_weaponclass"					"broad"
 
 		// Damage - When Used by Players
-		"damage_near_value"   							"15" // CHANGE/NERF: 20 -> 15 (5 -> 7 bullets)
-		"damage_far_value"								"9" // CHANGE/NERF: 12 -> 9 (9 -> 12 bullets)
+		"damage_near_value"   							"14" // CHANGE/NERF: 20 -> 14 (5 -> 8 bullets)
```

Comment:
I also think this should be 13 because 15 was pretty strong still

```suggestion
		"damage_near_value"   							"13" // CHANGE/NERF: 20 -> 13 (5 -> 8 bullets)
```